pipelines/build/prTester/README.md

```diff
@@ -0,0 +1,125 @@
+
+## How they're used
+
+The pr testers have their own admin and white lists attached to them on Jenkins.
+If you are on the whitelist or admin list, the pr tester will run against your pr whenever you make one and it will also allow you access to various commands you can run on your own PR or on someone else's:
+
+### `run tests`
+
+- Executes new [openjdk-build-compile-groovy](#openjdk-build-compile-groovy) and [openjdk-build-pr-tester](#openjdk-build-pr-tester) jobs against this PR. These jobs will populate the GitHub status checks field as described above. Please be patient as the [tester does not run concurrently yet](https://github.com/AdoptOpenJDK/openjdk-build/issues/2053) so it may take some time to execute the jobs if there is a long job queue. You can track the progress of it in [Jenkins](https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/) OR look at the status check message:
+
+  - Example of a pr that is in the queue:
+  ![Image of queued tester](../../../docs/images/pr_tester_queued.png)
+  - One that is at the front of the queue and currently being tested:
+  ![Image of building tester](../../../docs/images/pr_tester_building.png)
+
+- When the tester is done, it will return a response comment to the pr with feedback on the testing similar to the following:
+![Image of test result](../../../docs/images/pr_tester_result.png)
+
+- The message will vary depending on the result of the test. Please remember however, that failed tests may be due to existing problems in the nightly builds. If you're unsure if the tests failed because of your changes or not, check our [issue board](https://github.com/AdoptOpenJDK/openjdk-build/issues) and our [triage doc](https://docs.google.com/document/d/1vcZgHJeR8rW8U8OD23Uob7A1dbLrtkURZUkinUp7f_w/edit?usp=sharing) for the existing error
+
+  - 🟢**SUCCESS** 🟢 All the tests and jobs passed, congratulations!
+  - 🟠**FAILURE** 🟠 Some of the tests or jobs failed OR the job was aborted. Check the link in the field at the bottom of the PR for the job link to see exactly where it went wrong.
+  - 🔴**ERROR** 🔴 Something more serious went wrong with the tester itself. Please raise an issue with a link to the job, the error encountered and your PR that caused it (again, you can use the link at the bottom to see exactly what happened).
+
+- NOTE: [openjdk-build-compile-groovy](#openjdk-build-compile-groovy) does not produce these status messages for the reasons stated in [this issue](https://github.com/AdoptOpenJDK/openjdk-build/issues/2055#issuecomment-688802783).
+
+### `add to whitelist`
+
+- **ADMIN COMMAND ONLY**
+- This command adds a new user to the whitelist but not to the admin list. As of typing this, there is [currently no way to check if you have the correct permissions](https://github.com/AdoptOpenJDK/openjdk-build/issues/2055#issuecomment-688801090) to execute this command (or even run the tester yourself) so in absence of that, here is the current list of admins and whitelisted users:
```

Comment:
I can understand the reason for putting these lists in here, but I'm concerned that it could easily become out of date. Can we at the very least mention that the master lists are stored within the configuration of the https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/job/openjdk-build-pr-tester job? I would also say something like "If you're not sure if you're on the list, try it and if it doesn't work, ask to be added"